### PR TITLE
Sara

### PR DESCRIPTION
**Pull request by saranatour1**
added the models to the models .

### CODE REVIEW

File: .metadata/.log
Comment (Yousef-labadi) [inline .metadata/.log:1]: Must delete meta data folder plz

File: .metadata/.plugins/com.genuitec.eclipse.devstyle/recent.json
Comment (Yousef-labadi) [inline .metadata/.plugins/com.genuitec.eclipse.devstyle/recent.json:1]: also this

File: Feedme/src/main/java/com/codingdojo/feedme/models/Category.java
Comment (Yousef-labadi) [inline Feedme/src/main/java/com/codingdojo/feedme/models/Category.java:1]: Good sara and husam

File: Feedme/src/main/java/com/codingdojo/feedme/models/Resturant.java
Comment (Yousef-labadi) [inline Feedme/src/main/java/com/codingdojo/feedme/models/Resturant.java:1]: excelent Husam

**Review by Yousef-labadi — COMMENTED** [review]
reviwe the Models

**Comment by Yousef-labadi** [comment]
Good Team   💯

**Comment by saranatour1** [comment]
I removed the .metadata Folder through the .gitignore file and used to remove cashed versions.
```bash 
git rm -r --cached .metadata
git rm -r --cached .vscode
```

**Pull request closed without merging**